assets/tests/components/detours/diversionPage.saveDetour.test.tsx

```diff
@@ -83,7 +83,7 @@ describe("DiversionPage autosave flow", () => {
     })
 
     await waitFor(() => {
-      expect(putDetourUpdate).toHaveBeenCalledTimes(3)
+      expect(putDetourUpdate).toHaveBeenCalledTimes(5)
```

Comment:
not quite sure why this changed, I tried to figure out if there was a way to do "at least 2 times" but didn't see anything.

Reply:
okay actually I think this changed due to doing `                    "set.nearest-intersection-fallback"` in `Pick Start Point`